erc20-bridge-token/contracts/SelectivePausableUpgradable.sol

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+/**
+ * @dev Contract module which allows children to implement an emergency stop
+ * mechanism that can be triggered by an authorized account.
+ *
+ * This module is used through inheritance. It will make available the
+ * modifiers `whenNotPaused` and `whenPaused`, which can be applied to
+ * the functions of your contract. Note that they will not be pausable by
+ * simply including this module, only once the modifiers are put in place.
+ */
+abstract contract SelectivePausableUpgradable is Initializable, ContextUpgradeable {
+    /**
+     * @dev Emitted when the pause is triggered by `account`.
+     */
+    event Paused(address account, uint flags);
+
+    bool private _paused; // Deprecated!
```

Comment:
I think we should remove the deprecated field? If I get it correctly, this is related to the layout, then we need to have an appropriate name and description.

Reply:
Ok, removed since we haven't deployed it yet on mainnet

Reply:
e16126cedc28081154f50cd4576fc46b0b1634fc